### PR TITLE
Add example code, update README.md

### PR DESCRIPTION
**Pull request by cgwalters**
A quick example is very useful for people.

### CODE REVIEW

**Review by lucab — APPROVED** [review]

**Review by jlebon — APPROVED** [review]

File: example/example.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	if err != nil {
+		return err
+	}
+
+	var fcosstable stream.Stream
+	err = json.Unmarshal(body, &fcosstable)
+	if err != nil {
+		return err
+	}
+	awsimages := fcosstable.Architectures[targetArch].Images.Aws
```
Comment (bgilbert) [inline example/example.go:39]: It'd be good to check that the `targetArch` exists within `Architectures`.

File: example/example.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	if awsimages == nil {
+		return fmt.Errorf("No %s AWS images in stream", targetArch)
+	}
+	fmt.Printf("%s\n", awsimages.Regions[region].Image)
```
Comment (bgilbert) [inline example/example.go:43]: Likewise with the `region`.

**Comment by cgwalters** [comment]
Updated :new:

File: example/example.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	}
+
+	var fcosstable stream.Stream
+	err = json.Unmarshal(body, &fcosstable)
+	if err != nil {
+		return err
+	}
+	var arch stream.Arch
+	var ok bool
+	if arch, ok = fcosstable.Architectures[targetArch]; !ok {
```
Comment (bgilbert) [inline example/example.go:41]: This is okay, but a more succinct way to write it is

```suggestion
	arch, ok := fcosstable.Architectures[targetArch]
	if !ok {
```

**Review by bgilbert — APPROVED** [review]

**Pull request merged**